.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "mavsdk-proto"]
 	path = proto
 	url = https://github.com/mavlink/MAVSDK-Proto.git
+[submodule "src/third_party/xz-embedded/submodule"]
+	path = src/third_party/xz-embedded/submodule
+	url = https://github.com/tukaani-project/xz-embedded.git
```

Comment:
I'd prefer to get the dependency in the same way like all others (except gtest):

https://github.com/mavlink/MAVSDK/blob/main/third_party/CMakeLists.txt
E.g. https://github.com/mavlink/MAVSDK/blob/main/third_party/jsoncpp/CMakeLists.txt

Which library is up to you.

Reply:
This ^. [Do not add dependencies as submodules](https://www.acarg.ch/posts/cmake-deps/).

Reply:
Alright, I changed to the system lib liblzma in https://github.com/mavlink/MAVSDK/pull/2272/commits/51758840389ad5f54b72eb210ec80b28f77ee802